concepts/index.md

```diff
@@ -6,29 +6,38 @@ layout: documentation
 
 # Concepts
 
-Eclipse SmartHome strictly differentiates between the physical view and the functional view on the system.
-While the physical view is required for setup, configuration, troubleshooting, etc., the functional side covers the information that matter to applications, such as user interfaces and automation logic.
+openHAB is built around two distinctly different views.
+The first is the physical view - devices, configurations, connections, etc.
+The second is the function view.
+This functional view consists of how information about the physical view is presented in User Interfaces, how automation operates on representations of physical devices, or how an action on a User Interface affects the software associated with the physical device it represents.
+It can be helpful to know up front that these two views exist as you are learning about openHAB.
 
 ## Things, Channels, Items and Links
 
-**Things** are the entities that can be physically added to a system and which can potentially provide many functionalities at once. It is important to note that things do not have to be devices, but they can also represent a web service or any other manageable source of information and functionality.
-Things provide their functionality through a set of **Channels**. Channels are "passive" and can be regarded as a declaration of a Thing, what it can offer. It is up to the individual setup, which of the Channels are actively used through Items (see below).
+**Things** are the entities that can be physically added to a system.
+These Things may provide more than just one function (for example, a thermostat may allow you to set the room temperature and also turn on a fan).
```

Comment:
Just to think about:
Maybe this example is not that easy to grasp.
(What has the fan to do with that thermostat now? How does this help me?)

But as i said just to think about. I have no better example as suggestoin now, but will think about what a better one could be.

Reply:
@Confectrician here in the U.S., thermostats have a "Fan on/off" button as well as a set of buttons to adjust the set point.  I know in Europe, many of the thermostats do not have the fan switch, so I agree this is not a good example.

How about this:
"Things may provide more than just one function (for example, a z-wave multi-sensor may provide a motion detector and also measure room temperature)"

I actually have one of those...

Reply:
Hey Brad,

Suggestion sounds good and I think it is easy to understand for everyone.

I am perfectly fine with it.
Good idea.